src/Components/WebAssembly/WebAssembly.Authentication/src/Services/DeserializedAuthenticationStateProvider.cs

```diff
@@ -5,6 +5,7 @@
 using System.Security.Claims;
 using Microsoft.AspNetCore.Components.Authorization;
 using Microsoft.Extensions.Options;
+using static Microsoft.AspNetCore.Internal.LinkerFlags;
```

Comment:
Where was this used?

Reply:
That's where `JsonSerialized` is defined. I might have just used `LinkerFlags.JsonSerialized`, but this `using static` pattern is consistently used elsewhere in the project where `JsonSerialized` is used.